<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GSettingsDialog</class>
 <widget class="QDialog" name="GSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="0">
    <widget class="QListWidget" name="m_listSections">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="currentRow">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Game</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Appearance</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="m_stackedSections">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_pageGame">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>517</width>
            <height>358</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="title">
              <string>Game Configuration</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="toolTip">
                 <string>Indicates maximum count of continuously playable non-jumping moves. If this number is reached during the game, then game ends.</string>
                </property>
                <property name="text">
                 <string>Maximum number of moves without jump</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::NoTextInteraction</set>
                </property>
                <property name="buddy">
                 <cstring>m_spinMoves</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="GSpinBox" name="m_spinMoves">
                <property name="wrapping">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::PlusMinus</enum>
                </property>
                <property name="keyboardTracking">
                 <bool>false</bool>
                </property>
                <property name="suffix">
                 <string> moves</string>
                </property>
                <property name="minimum">
                 <number>30</number>
                </property>
                <property name="maximum">
                 <number>80</number>
                </property>
                <property name="singleStep">
                 <number>2</number>
                </property>
                <property name="value">
                 <number>60</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Strategy for &quot;Find Best Move&quot; function</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="m_comboStrategy">
                <item>
                 <property name="text">
                  <string>Standard</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Advanced</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QGroupBox" name="groupBox_7">
                <property name="title">
                 <string>Starting Player</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QRadioButton" name="m_radioWhiteStarts">
                   <property name="text">
                    <string>White Player</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="m_radioBlackStarts">
                   <property name="text">
                    <string>Black Player</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="3" column="0" colspan="2">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QGroupBox" name="groupBox_4">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>White Player</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_5">
                   <item row="1" column="0">
                    <widget class="QGroupBox" name="m_groupWhite">
                     <property name="title">
                      <string>Difficulty</string>
                     </property>
                     <layout class="QGridLayout" name="gridLayout_6">
                      <item row="0" column="0">
                       <widget class="QRadioButton" name="m_whiteEasy">
                        <property name="text">
                         <string>Easy</string>
                        </property>
                        <property name="checked">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="0">
                       <widget class="QRadioButton" name="m_whiteMedium">
                        <property name="text">
                         <string>Medium</string>
                        </property>
                       </widget>
                      </item>
                      <item row="2" column="0">
                       <widget class="QRadioButton" name="m_whiteHard">
                        <property name="text">
                         <string>Hard</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1" rowspan="3">
                       <widget class="QLabel" name="m_labelWhite">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>34</height>
                         </size>
                        </property>
                        <property name="text">
                         <string notr="true"/>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignCenter</set>
                        </property>
                        <property name="textInteractionFlags">
                         <set>Qt::NoTextInteraction</set>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QComboBox" name="m_comboWhite">
                     <item>
                      <property name="text">
                       <string>Human</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Computer</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_5">
                  <property name="title">
                   <string>Black Player</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_7">
                   <item row="1" column="0">
                    <widget class="QGroupBox" name="m_groupBlack">
                     <property name="title">
                      <string>Difficulty</string>
                     </property>
                     <layout class="QGridLayout" name="gridLayout_8">
                      <item row="0" column="0">
                       <widget class="QRadioButton" name="m_blackEasy">
                        <property name="text">
                         <string>Easy</string>
                        </property>
                        <property name="checked">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="0">
                       <widget class="QRadioButton" name="m_blackMedium">
                        <property name="text">
                         <string>Medium</string>
                        </property>
                       </widget>
                      </item>
                      <item row="2" column="0">
                       <widget class="QRadioButton" name="m_blackHard">
                        <property name="enabled">
                         <bool>true</bool>
                        </property>
                        <property name="text">
                         <string>Hard</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="1" rowspan="3">
                       <widget class="QLabel" name="m_labelBlack">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>34</height>
                         </size>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignCenter</set>
                        </property>
                        <property name="textInteractionFlags">
                         <set>Qt::NoTextInteraction</set>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QComboBox" name="m_comboBlack">
                     <item>
                      <property name="text">
                       <string>Human</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Computer</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="4" column="0" colspan="2">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>403</width>
                  <height>136</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pageAppearance">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>145</width>
            <height>198</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Appearance Configuration</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_6">
                <property name="title">
                 <string>Control Box Alignment</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_11">
                 <item row="0" column="0">
                  <widget class="QRadioButton" name="m_radioControlLeft">
                   <property name="text">
                    <string>Left</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QRadioButton" name="m_radioControlRight">
                   <property name="text">
                    <string>Right</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="2" column="0">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QGroupBox" name="groupBox_8">
                <property name="title">
                 <string>Application Language</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QListWidget" name="m_listLanguages">
                   <property name="alternatingRowColors">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pageOther">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>115</width>
            <height>16</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Other Configuration</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="m_buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GSpinBox</class>
   <extends>QSpinBox</extends>
   <header>gspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_listSections</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>m_stackedSections</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
